backend/backend-python/debug_framework/tests/test_idempotent_schema.py

```
@@ -0,0 +1,58 @@
+        # Verify that tables exist and work correctly after multiple runs
+        print("\nTest 4: Verifying functionality after multiple schema runs...")
+        cursor.execute("INSERT INTO debug_sessions (model_path) VALUES (?)", ("/test/model",))
+        cursor.execute("SELECT COUNT(*) FROM debug_sessions")
+        count = cursor.fetchone()[0]
+        print(f"✓ Tables functional after multiple runs (session count: {count})")
+
+        conn.commit()
+        conn.close()
+
+        print("\n✅ Idempotency test passed!")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)